### ISSUE TITLE
DeepHit: A Deep Learning Approach to Survival Analysis with Competing Risks

### ISSUE THREAD

**Issue by cmclean5** [cmclean5/PublicHealthModels#1]
## TL;DR
This paper proposes a deep neural network approach to survival analysis.

### Paper Link
http://medianetlab.ee.ucla.edu/papers/AAAI_2018_DeepHit

### Author/Institution

Changhee Lee (University of California), William R. Zame (University of California),  
Jinsung Yoon (University of California), Mihaela van der Schaar (University of California, University of Oxford, Alan Turing Institute)

## Overview

This paper proposes a deep neural network approach to survival analysis, to learn the distribution of survival times directly. The DeepHit model makes no assumptions about the underlying stochastic process and allows for the possibility that the relationship between covariates and risk(s) changes over time. The DeepHit model can handle competing risks; i.e. settings in which there is more than one possible event of interest.

## Methods

Survival analysis, neural networks.

## Comments

Python
